spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -489,13 +486,9 @@ public SparkContext createSparkContext_1() {
     if (execUri != null) {
       conf.set("spark.executor.uri", execUri);
     }
-    if (System.getenv("SPARK_HOME") != null) {
-      conf.setSparkHome(System.getenv("SPARK_HOME"));
```

Comment:
`SparkInterpreter.java` runs in interpreter process which is launched by `interpreter.sh`. That means SPARK_HOME should already be set before `SparkInterpreter` is launched. So it is user's responsibility to set `SPARK_HOME` either in `zeppelin-env.sh` or interpreter setting.